pyth_observer/events.py

```diff
@@ -185,23 +185,33 @@ class PriceDeviation(PriceValidationEvent):
 
     def is_valid(self) -> bool:
         delta = self.publisher_aggregate.price - self.price.aggregate.price
-        self.deviation = abs(delta / self.price.aggregate.price) * 100
-
-        if (self.price.is_publishing(self.publisher_key) and
-                self.price.is_aggregate_publishing() and
-                self.deviation > self.threshold):
+        if self.price.aggregate.price == 0:
+            #TODO: add another alert that validates whether the aggregate price is close to the truth
             return False
+        else:
+            self.deviation = abs(delta / self.price.aggregate.price) * 100
+
+            if (self.price.is_publishing(self.publisher_key) and
+                    self.price.is_aggregate_publishing() and
+                    self.deviation > self.threshold):
+                return False
```

Comment:
```suggestion
        self.deviation = abs(delta / self.price.aggregate.price) * 100

        if (self.price.is_publishing(self.publisher_key) and
                self.price.is_aggregate_publishing() and
                self.deviation > self.threshold):
            return False
```

There is no possible way code can execute after a return, so the else is just an extra indent (which sacrifices readability) for no good reason.